www/front_src/src/Resources/columns/State/DetailsTable/index.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+
+import format from 'date-fns/format';
+import parseISO from 'date-fns/parseISO';
```

Comment:
Does it break bundle size if we import like this ?
```suggestion
import { format, parseISO } from 'date-fns/format';
```

Reply:
yes, the date-fns package doesn't import using ESM